data.py

```diff
@@ -195,24 +197,33 @@ def load_query_string(self, data: str) -> None:
 
 class DataV3(Data):
     @staticmethod
-    def get_topics() -> List[topics.Topic]:
+    def get_topics(self) -> List[topics.Topic]:
```

Comment:
Not sure if this runs for you, but this doesn't work for me because `self` is a parameter but it is called elsewhere without `self` being passed. (This is a static method, see [here](https://pythonbasics.org/static-method/))

To fix this, remove the `self` parameter, make `create_voltage_topic` also static, and call it with `DataV3.create_voltage_topic()`.